### ISSUE TITLE
Feedback: index.html

### ISSUE THREAD

**Issue by elliotmjackson** [bhammill9/luigis#2]
The following contains feedback on [index.html](https://github.com/bhammill9/luigis/blob/main/index.html)

**Positive Aspects**:
1. The DOCTYPE declaration is correctly used to specify the document type and version.
2. Good use of meta tags for character encoding and viewport settings.
3. The `<link>` tags in the `<head>` section have been correctly used for stylesheets, favicon, and font imports.
4. The structure of the code is clean, making it easy to read.
5. Comments are used to section out parts of the webpage, aiding readability.
6. Image sources are relative, indicating an organized file structure.

**Areas of Improvement / Suggestions**:
1. **Navigation Links**:
    - The list items under the `#menu_items` ID (Home, Menu, Contact) are not wrapped in anchor tags (`<a>`). If these are supposed to be clickable links, they should have `<a>` tags around them.
    
    ```html
    <ul>
      <li><a href="#home">Home</a></li>
      <li><a href="#menu">Menu</a></li>
      <li><a href="#contact">Contact</a></li>
    </ul>
    ```

2. **Image alt Attributes**:
    - All the `<img>` tags are missing the `alt` attribute. This attribute provides a text alternative for images, which is essential for accessibility and SEO.
    
    ```html
    <img src="./resources/images/icons8-pizza-80.png" alt="Pizza Icon">
    ```

3. **Consistency in Class Naming**:
    - The class naming seems inconsistent. While some classes describe content (`banner_content`, `team_content`), others describe the container (`pizza_container`, `team_container`). Maintaining a consistent naming convention can help during the CSS styling process and make the code more readable.

4. **Semantic HTML**:
    - Consider using more semantic elements like `<header>`, `<section>`, `<article>`, `<footer>`, etc., which can make the structure clearer and more accessible.
    
    For example, the header could be written as:
    
    ```html
    <header>
      ...contents of header section...
    </header>
    ```

5. **Redundant Containers**:
    - Some sections, like the pizza types or team members, seem to have an extra layer of containers (like `pizza_center` or `member_center`). While these may have a purpose in styling, ensure they are necessary before finalizing the design.

6. **Typographical Errors**:
    - There's a small typo in the "About" section: "renound" should be "renowned".

7. **Contact Section**:
    - For better user experience, consider turning the phone number and email into clickable links using the `tel:` and `mailto:` protocols.

    ```html
    <li><a href="tel:0400000000">0400 000 000</a></li>
    <li><a href="mailto:luigi@luigispizza.com">luigi@luigispizza.com</a></li>
    ```

In summary, while the code gives a clear structure to a webpage and the user has a basic understanding of HTML, there are areas of improvement in terms of accessibility, semantics, and user experience. Incorporating these suggestions will lead to a more polished and professional result.
